src/Domains/Connectors/Jobs/MailCaddieLabJob.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Connectors\Jobs;
```

Comment:
@FredPeal Connector\Internal , because this are Notifications workflow, actions, jobs we are doing for ourselfs